agent/api/task/task.go

```diff
@@ -299,8 +299,7 @@ type Task struct {
 
 	NetworkNamespace string `json:"NetworkNamespace,omitempty"`
 
-	// TODO: Will need to initialize/set the value in a follow PR
-	FaultInjectionEnabled bool `json:"FaultInjectionEnabled,omitempty"`
+	EnableFaultInjection bool `json:"enableFaultInjection,omitempty"`
```

Comment:
nit: we should follow the same syntax in terms of JSON field naming. (i.e. EnableFaultInjection)

Reply:
Discussed offline, leaving as `enableFaultInjection`